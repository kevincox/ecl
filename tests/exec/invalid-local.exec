cmd: load invalid-local.ecl
status: 1
