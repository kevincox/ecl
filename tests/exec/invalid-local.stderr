Failed to parse "invalid-local.ecl": ParseError { typ: Unexpected(1.11, Add, []), msg: "parsing local var" }

Error occured.
